rq/job.py

```diff
@@ -609,9 +606,12 @@ def cleanup(self, ttl=None, pipeline=None, remove_from_queue=True):
             self.delete(pipeline=pipeline, remove_from_queue=remove_from_queue)
         elif not ttl:
             return
-        elif ttl > 0:
+        else:
             connection = pipeline if pipeline is not None else self.connection
-            connection.expire(self.key, ttl)
+            if ttl > 0:
+                connection.expire(self.key, ttl)
+            else:
+                connection.persist(self.key)
```

Comment:
That's about the line in the docstring for `cleanup`: [If ttl is negative, don't set an expiry to it (persist
          forever)](https://github.com/katichev/rq/blob/explicit_cleanup/rq/job.py#L602)
Before, `cleanup(ttl=-1)` did not really clear the expiration time to unlimited.

The idea was to manage the key expiration in one single place. Now I see that I forget to replace `persist` with `cleanup` [here](https://github.com/katichev/rq/blob/explicit_cleanup/rq/job.py#L554). :)

Anyway I need another one commit. I have two options here: 1. Revert back the change you've mentioned; 2. replace `persist` with `cleanup(ttl=-1)`. What do you suggest?